src/components/NavBarConsumerHome.js

```diff
@@ -0,0 +1,77 @@
+import '../styles/NavBar.css';
+import LogOutButton from './LogOutButton';
+import React, { useState } from 'react';
+
```

Comment:
If you notice the NavBars are similar, its because they are meant to be. all of them use the same general style, however each page needs different buttons and options on its Nav bar, and as the app grows the differences will be more and more apparent, right now there are just a few differences per page. if there is a cleaner way to handle this let me know

Reply:
okay leave for now

Reply:
ex

<NavBar header="myHeader" list={[{buttonName: x,  href='/blah'}, {buttonName: x2,  href='/blah2'}]}>

const navbar = () => {
  <div>
    {header}
    <ul>
      {list.map((button) => <li>{button.buttonName}</li>)}
    <ul>
  </div>

}